docs/setup/agent/docker-compose.mdx

```diff
@@ -6,9 +6,10 @@ import CodeBlock from '@theme/CodeBlock';
 import DskAgentDockerCompose from '!!raw-loader!../../deployments/dsk-agent/docker-compose.yml';
 import DskAgentEnv from '!!raw-loader!../../deployments/dsk-agent/sample.env';
 import DockerComposeAutomatedSetup from './_docker-compose-automated-setup.md';
+import ApiCredentials from '../../common/\_api-credentials.md';
```

Comment:
\ needed?

Reply:
also work's without backslash. removed it.